lib/user_space/user_x86_64_atomic.h

```diff
@@ -78,29 +87,60 @@ static inline void m0_atomic64_inc(struct m0_atomic64 *a)
  */
 static inline void m0_atomic64_dec(struct m0_atomic64 *a)
 {
+#ifdef CONFIG_X86_64
 	asm volatile("lock decq %0"
 		     : "=m" (a->a_value)
 		     : "m" (a->a_value));
+#elif defined CONFIG_AARCH64 /*aarch64*/
+	m0_atomic64_sub(a, (int64_t)1);
+#endif
 }
 
 /**
    Atomically adds given amount to a counter
  */
 static inline void m0_atomic64_add(struct m0_atomic64 *a, int64_t num)
 {
+#ifdef CONFIG_X86_64
 	asm volatile("lock addq %1,%0"
 		     : "=m" (a->a_value)
 		     : "er" (num), "m" (a->a_value));
+#elif defined CONFIG_AARCH64 /*aarch64*/
```

Comment:
the comment /*aarch64*/ doesn't give more info than CONFIG_AARCH64, I think the comment can be removed. Same as other places.

Reply:
Since comment  add more documentation. Let it be like that

Reply:
It does not add any useful information here. It only adds an excess noise.